x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/session/IndexResolverFieldNamesTests.java

```diff
@@ -1364,6 +1365,7 @@ public void testMetrics() {
     }
 
     public void testLookupJoin() {
+        assumeTrue("LOOKUP JOIN available as snapshot only", EsqlCapabilities.Cap.LOOKUP_V4.isEnabled());
```

Comment:
Surely the capability should be JOIN_LOOKUP_V6 (or even higher if more work gets in before this PR).

Reply:
```suggestion
        assumeTrue("LOOKUP JOIN available as snapshot only", EsqlCapabilities.Cap.JOIN_LOOKUP_V6.isEnabled());
```